modules/weboramaRtdProvider.js

```diff
@@ -689,19 +697,8 @@ class WeboramaRtdProvider {
     /** @type {string} */
     const bidder = bidderAliasRegistry[bid.bidder] || bid.bidder;
 
-    switch (bidder) {
-      case 'appnexus':
-        this.#handleAppnexusBid(bid, profile);
-        break;
-      case 'pubmatic':
-        this.#handlePubmaticBid(bid, profile);
-        break;
-      case 'smartadserver':
-        this.#handleSmartadserverBid(bid, profile);
-        break;
-      case 'rubicon':
-        this.#handleRubiconBid(bid, profile, metadata);
-        break;
+    if (bidder === APPNEXUS_BIDDER) {
```

Comment:
To summarize what I think is missing:

 - appnexus (and clones) now parse keywords as `key=value` pairs. so this should serialize objects like `{ webo_ctx: ["A", "B"], webo_ds: ["C", "D"]}` down to a single string such as `webo_ctx=A,webo_ctx=B,webo_ds=C,webo_ds=D`.
​
- because appnexus has clones, this should ideally not check for `appnexus` specifically. You could take the list of bidders to set keywords for as config, or you  could even set these keywords on all the bidders you want to set segments for (non-appnexus bidders will probably have no use for them but they shouldn't hurt).